media/css/_navigation.scss

```diff
@@ -0,0 +1,122 @@
+.navbar-inner {
+  background: $sidebar-background;
+  border: 0;
+}
+
+.navbar-inverse {
+  .navbar-inner {
+    background: $sidebar-background;
+  }
+}
+
+.navbar {
+  .brand {
+    margin-left: 3px;
+    margin-right: 3px;
+    max-height: 50px;
+    overflow: visible;
+    padding: 0 3px;
+
+    &:hover {
+      background: $sidebar-link-highlight;
+    }
+  }
+
+  .nav {
+    display: inline;
+    font-size: 14px;
+    list-style: none;
+    margin: 0;
+    text-align: left;
+
+    li {
+      &:hover {
+        background: $sidebar-link-highlight;
+        color: $white;
+      }
+    }
+
+    span {
+      cursor: pointer;
+      display: inline-block;
+      height: 20px;
+      padding: 15px;
+    }
+
+    .active {
+      a {
+        background: $sidebar-link-highlight;
+
+        &:hover,
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 3, but was 4